architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,86 @@
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Authorization determines whether an identity is allowed to perform any action.
+Note that authorization is a separate step from
+link:authentication.html[authentication], which is more about determining the
+identity of who is taking the action.
+
+Authorization is managed using:
+
+[horizontal]
+Rules:: Describe sets of permitted verbs on a set of resource types.
+Roles:: Collection of rules.
+Bindings:: Associate users and/or groups with a role.
+
+Users and groups may be bound to multiple roles at the same time.
+
+Several factors are combined to make the decision when authorization is
+evaluated:
+
+[horizontal]
+Identity:: Both the username and list of groups the user belongs to.
+Action:: The action being performed. In most cases, this consists of:
+Project::: The project being accessed.
+Verb::: Can be `get`, `list`, `create`, `update`, or others.
+Resource::: The API endpoint being accessed. This is distinct from the resource
+being touched, such as pods, `*deploymentconfigs*`, or builds.
+Bindings:: The full list of bindings.
+
+The evaluation goes through the following steps:
+
+. Use the identity and the `action.project` to find all bindings that apply to the user or their groups.
+. Use the bindings to locate all the roles that apply.
+. Use the roles to find all the rules that apply.
+. Check the action against each rule to find a match.
+. If no matching rule is found, the action is then denied by default.
+
+You can link:../../admin_guide/manage_authorization_policy.html[manage
+authorization policy] using the administrator CLI.
+
+== Roles
+OpenShift includes four global roles by default:
+
+cluster-admin:: A super-user that can perform any action in any project.
+
+admin:: A project manager. If used in a link:#global-roles-and-bindings[local
+binding], an *admin* user will have rights to view any resource in the project
+and modify any resource in the project except for role creation and quota.  If
+the *cluster-admin* wants to allow an *admin* to modify roles, the
+*cluster-admin* must create a project scoped Policy object.
+
+edit:: A user that can modify most objects in a project, but does not have the
+power to view or modify roles or bindings.
+
```

Comment:
I think we should use a table here, which would be much more clear than a list as we've discussed in the past.